### ISSUE TITLE
Download Deck in OCTGN format

### ISSUE THREAD

**Issue by sacrelicious2** [Kamalisk/arkhamdb#32]
It would be nice to be able to download a deck in OCTGN format, compatible with the arkham horror module https://github.com/GeckoTH/arkham-horror

**Comment by Kamalisk** [comment]
If they have it working, then I just need the octgn card ids from somewhere.

**Comment by sacrelicious2** [comment]
It looks like the card ids (at least for the core set) are stored here: https://github.com/GeckoTH/arkham-horror/blob/master/o8g/Sets/Core%20Set/set.xml.  Curse of the Rougarou cards in a similar named location.  Example deck format can be seen here: https://github.com/GeckoTH/arkham-horror/blob/master/o8g/Decks/01%20-%20Core/Starter%20-%20Agnes%20Baker.o8d.  Note the 2 investigator cards, one for the main card, one for the mini used at locations.

**Comment by Kamalisk** [comment]
I added the octgn ids, just need to know the format it expects for the deck file, and how to do these tiny investigator card things.

**Comment by Kamalisk** [comment]
it has been added, for the core set at least, decks seem valid but it is missing the tiny investigator id, assuming that is needed, otherwise it should work.

**Comment by rushl** [comment]
I just submitted a pull request to bring the export into alignment with the OCTGN module. The export is actually currently including the investigator mini card, but not the actual investigator card. (At least, that's what happened when I tried an export.) I'm not familiar enough with the arkhamdb.com code to find and fix that issue, but I can help with the IDs that are needed. 

Basically, investigator cards come in pairs. One of the XML nodes has the attribute size="InvestigatorCard" with a card property of "Type" set to "Investigator". That's the actual investigator card. The other has the attribute size="MiniCard" with a card property of "Type" set to "Mini". For an exported deck to be usable directly in a game, the deck file must contain both of those cards in the "Investigator" section.
